### PR TITLE
feat: homepage file cleaned up

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- homepage file cleaned up

@mcstover should we remove support for New Yeah Share Campaign?

### CODE REVIEW

File: src/pages/Homepage/Homepage.vue [hunk 1]
```diff
@@ -4,24 +4,11 @@
 
 <script>
 /* eslint-disable vue/multi-word-component-names */
-import { gql } from '@apollo/client';
-import experimentAssignmentQuery from '@/graphql/query/experimentAssignment.graphql';
-import experimentVersionFragment from '@/graphql/fragments/experimentVersion.graphql';
 import { preFetchAll } from '@/util/apolloPreFetch';
 import numeral from 'numeral';
 
 const ContentfulPage = () => import('@/pages/ContentfulPage');
```
Comment (dyersituations) [inline src/pages/Homepage/Homepage.vue:10]: Can we remove this file entirely?

Comment (roger-in-kiva) [reply src/pages/Homepage/Homepage.vue:10]: This from a slack convo with Stover:

> It should be safe to clean up the old commented out route (leave the redirect in place for now) followed by cleaning up the Homepage.vue file and any homepage specific (non-contentful) components.
> We should keep the Contentful page intact for now

https://kivahq.slack.com/archives/CB7GRAFKK/p1700513842600929

Comment (mcstover) [reply src/pages/Homepage/Homepage.vue:10]: I don't think we should remove the Contentful page and related code just yet. But I think it's fine to remove the Homepage.vue file...

Comment (roger-in-kiva) [reply src/pages/Homepage/Homepage.vue:10]: Gotcha, thanks!

File: src/components/Homepage/MonthlyGoodExplained.vue [hunk 2]
```diff
@@ -1,162 +0,0 @@
...
-				>
-					Learn more
-				</router-link>
-			</div>
-		</div>
-	</div>
-</template>
-
-<script>
-import MgStepOne from '@/assets/inline-svgs/hp-mg-1-signup.svg';
```
Comment (dyersituations) [inline src/components/Homepage/MonthlyGoodExplained.vue:73]: Remove these assets if they aren't used elsewhere (looks like they aren't?)

Comment (roger-in-kiva) [reply src/components/Homepage/MonthlyGoodExplained.vue:73]: Thanks!

**Review by dyersituations — APPROVED** [review]

**Pull request merged**